app/views/handlers/feedback.py

```diff
@@ -85,38 +86,42 @@ def get_page_title(self) -> str:
 
     def handle_post(self) -> None:
         session_data = self._session_store.session_data
-        session_data.feedback_count += 1
-
-        feedback_metadata = FeedbackMetadata(
-            session_data.case_id,
-            session_data.tx_id,
-        )
-
-        # pylint: disable=no-member
-        # wtforms Form parents are not discoverable in the 2.3.3 implementation
-        feedback_message = FeedbackPayload(
-            metadata=self._questionnaire_store.metadata,
-            response_metadata=self._questionnaire_store.response_metadata,
-            schema=self._schema,
-            case_id=session_data.case_id,
-            submission_language_code=session_data.language_code,
-            feedback_count=session_data.feedback_count,
-            feedback_text=self.form.data.get("feedback-text"),
-            feedback_type=self.form.data.get("feedback-type"),
-        )
-        message = feedback_message()
-        metadata = feedback_metadata()
-        message.update(metadata)
-        encrypted_message = encrypt(
-            message, current_app.eq["key_store"], KEY_PURPOSE_SUBMISSION  # type: ignore
-        )
+        if isinstance(session_data, SessionData):
```

Comment:
We should avoid this indenting for the sake of type hinting. We should escape early but the check here should be needed if we escape early on _init_ if we don't have session data.

Reply:
So, as we discussed, this will get type ignore for now and we will revisit once we decide about the approach.